test/runtests.jl

```diff
@@ -46,6 +46,17 @@ end
     @test max(eigenvalues...) ≈ 7.634155067582592
 end
 
+# ------------------------------------------------------------------------------
+# SUPG advection operator example with conformal maps
+# ------------------------------------------------------------------------------
+
+@testset "  ex011: SUPG advection example               " begin
```

Comment:
```suggestion
@testset "  ex011: SUPG advection example              " begin
```
woops, off by one with my suggestion